go/client/color.go

```diff
@@ -92,6 +92,16 @@ func colorByteSequence(code int) []byte {
 func (cp CodePair) OpenBytes() []byte  { return colorByteSequence(cp.Open) }
 func (cp CodePair) CloseBytes() []byte { return colorByteSequence(cp.Close) }
 
+func HasColor() bool {
```

Comment:
Note this function was originally here before being split off for Windows, which is no longer necessary